components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -79,6 +83,10 @@ const ContributorHighlightCard = ({
 
   const { follow, unFollow, isError } = useFollowUser(user);
 
+  const { data: emojis, getEmojiNameById } = useFetchAllEmojis();
```

Comment:
This should be moved outside of this component to the page itself and passed in as a prop so its only fetched/shared once for each highlight card on the page.

Reply:
When you say move outside, Do you mean it for all three reactions-related hooks or just the `useFetchEmojis`?